src/components/LabEditorGUI/SelectSwitch.tsx

```diff
@@ -11,6 +11,7 @@ import TwentyFourPort from '../../assets/images/LabEditor/Twenty-FourPortSwitch.
 import FortyEightPort from '../../assets/images/LabEditor/Forty-EightPortSwitch.png';
 import closeIcon from '../../assets/icons/close.svg';
 import styles from './SelectSwitch.module.scss';
+import SwitchButton from './SwitchButton';
 
 const SelectSwitch = ({close, addSwitch}: any) => {
```

Comment:
Is it possible to move this constant to the component? or is there a reason it needs to be in this scope?

Reply:
I think that is needed because it is what imports the SwitchButton component. I'm not sure if there is another way to import components.

Reply:
> Is it possible to move this constant to the component? or is there a reason it needs to be in this scope?

I'm unsure what you're asking, this is how imports work?